### PR TITLE
[lit-next] Fix typings to match module/main

### PR DESCRIPTION
**Pull request by Artur-**
Fixes https://github.com/Polymer/lit-element/issues/1164

### CODE REVIEW

File: packages/lit-element/package.json [hunk 1]
```diff
@@ -93,7 +93,7 @@
     "tslib": "^2.0.3",
     "typescript": "^4.1.3"
   },
-  "typings": "lit-element.d.ts",
+  "typings": "index.d.ts",
```
Comment (justinfagnani) [inline packages/lit-element/package.json:96]: Do we even need this field anymore? I feel like TypeScript looks here by default now.

Comment (Artur-) [reply packages/lit-element/package.json:96]: https://www.typescriptlang.org/docs/handbook/declaration-files/publishing.html says

"Also note that if your main declaration file is named index.d.ts and lives at the root of the package (next to index.js) you do not need to mark the "types" property, though it is advisable to do so."

Comment (Artur-) [reply packages/lit-element/package.json:96]: It does seems to work also if this is deleted

Comment (justinfagnani) [reply packages/lit-element/package.json:96]: Let's keep it, but it does like like `"types"` is the newer field.

Comment (Artur-) [reply packages/lit-element/package.json]: Changed to 'types' and moved it next to other index files

**Comment by Artur-** [comment]
Anything more needed for this?

**Review by kevinpschaaf — APPROVED** [review]

**Comment by kevinpschaaf** [comment]
Thanks!

**Pull request merged**